test/chi/components/data-table.pug

```diff
@@ -0,0 +1,1347 @@
+            .chi-button__content
+              i.chi-icon.icon-chevron-up(aria-hidden='true')
+        .chi-data-table__cell(data-label='Name')
+          div Name 1
+        .chi-data-table__cell(data-label='ID')
+          div name-1
+        .chi-data-table__cell(data-label='Last Login')
+          div 18 Dec 2020 3:26 p.m. 
+      .chi-data-table__row-child.-p--2(role='row') Accordion content
+      .chi-data-table__row
+        .chi-data-table__cell.-expandable
+          button.chi-button.-icon.-flat.-sm(aria-label='Expand row')
+            .chi-button__content
+              i.chi-icon.icon-chevron-right(aria-hidden='true')
+        .chi-data-table__cell(data-label='Name')
+          div Name 2
+        .chi-data-table__cell(data-label='ID')
+          div name-2
+        .chi-data-table__cell(data-label='Last Login')
+          div 18 Dec 2020 2:38 a.m. 
+      .chi-data-table__row
+        .chi-data-table__cell.-expandable
+          button.chi-button.-icon.-flat.-sm(aria-label='Expand row')
+            .chi-button__content
+              i.chi-icon.icon-chevron-right(aria-hidden='true')
+        .chi-data-table__cell(data-label='Name')
+          div Name 3
+        .chi-data-table__cell(data-label='ID')
+          div name-3
+        .chi-data-table__cell(data-label='Last Login')
+          div 5 Nov 2020 10:15 a.m.
+    .chi-data-table__footer
+      nav.chi-pagination(role='navigation' aria-label='Pagination')
+        .chi-pagination__content
+          .chi-pagination__start
+            .chi-pagination__results
+              span.chi-pagination__label 6 results
+            .chi-pagination__page-size
+              .chi-dropdown
+                button#pagesize-9.chi-button.-flat.-px--1.chi-dropdown__trigger 20
+                .chi-dropdown__menu.-w--xs
+                  a.chi-dropdown__menu-item.-active(href='#') 20
+                  a.chi-dropdown__menu-item(href='#') 40
+                  a.chi-dropdown__menu-item(href='#') 60
+                  a.chi-dropdown__menu-item(href='#') 80
+                  a.chi-dropdown__menu-item(href='#') All
+              span.chi-pagination__label per page
+          .chi-pagination__center
+            .chi-pagination__button-group.chi-button-group
+              button.chi-button.-icon.-flat(aria-label='Previous page')
+                .chi-button__content
+                  i.chi-icon.icon-chevron-left(aria-hidden='true')
+              button.chi-button.-flat(aria-label='Page 1') 1
+              button.chi-button.-flat(aria-label='Page 2') 2
+              button.chi-button.-icon.-flat(aria-label='Next page')
+                .chi-button__content
+                  i.chi-icon.icon-chevron-right(aria-hidden='true')
+          .chi-pagination__end
+            .chi-pagination__jumper
+              label.chi-pagination__label(for='jumper-input-9') Go to page:
+              input#jumper-input-9.chi-input(type='text')
+
+.test-accordion-portal
```

Comment:
Portal test cases will be set up in a different way. Let's keep the file Lumen theme specific